src/tbp/monty/frameworks/environments/embodied_data.py

```diff
@@ -27,6 +27,7 @@
     SurfacePolicy,
 )
 from tbp.monty.frameworks.models.motor_system import MotorSystem
+from tbp.monty.frameworks.models.motor_system_state import MotorSystemState
```

Comment:
Why is this called `MotorSystemState`? Isn't it the state of the agent that is passed to the motor system since some policies rely on the agent state to decide the next action?